.env.example

```
@@ -1,6 +1,9 @@
 # RPC for the SEDA network you want to interact with
 SEDA_RPC_ENDPOINT=https://rpc.testnet.seda.xyz
 
+# (optional) Explorer for the SEDA network you want to interact with 
+SEDA_EXPLORER_URL=https://devnet.explorer.seda.xyz
```

Should be the same as the RPC endpoint.
```suggestion
SEDA_EXPLORER_URL=https://testnet.explorer.seda.xyz
```

---

do we want to default to testnet or devnet for these type of docs